app/tests/unit/components/fixtures/acronymUsers.json

```diff
@@ -0,0 +1,38 @@
+[
+  {
+    "keycloakGuid": "k1",
+    "userAcronym": {
+      "userAcronymId": "a",
+      "owner": false,
+      "createdAt": "2020-03-17T20:53:06.565Z",
+      "updatedAt": "2020-03-17T20:53:06.565Z",
+      "deletedAt": null,
+      "acronymId": "ccbb8b92-4a64-4c32-abb4-8cf5f15d7843",
+      "userId": "1"
+    }
+  },
+  {
+    "keycloakGuid": "k-not-there",
+    "userAcronym": {
+      "userAcronymId": "b",
+      "owner": false,
+      "createdAt": "2020-03-30T04:33:59.375Z",
+      "updatedAt": "2020-03-30T04:33:59.375Z",
+      "deletedAt": null,
+      "acronymId": "ccbb8b92-4a64-4c32-abb4-8cf5f15d7843",
+      "userId": "2"
+    }
+  },
+  {
+    "keycloakGuid": "k2",
+    "userAcronym": {
+      "userAcronymId": "c",
+      "owner": false,
+      "createdAt": "2020-03-30T04:33:59.375Z",
+      "updatedAt": "2020-03-30T04:33:59.375Z",
+      "deletedAt": null,
+      "acronymId": "ccbb8b92-4a64-4c32-abb4-8cf5f15d7843",
+      "userId": "3"
+    }
+  }
+]
```

Comment:
No need to change right now, but I'm half tempted to say given this data structure, we should consider having this be a generator function instead -> that way we can dynamically assign created, updated, and acronymId uuids on each unit test run, and test for equivalence per instance instead.

Reply:
Yeah could be more useful like that when needing more dynamic test results